pkg/root/trusted_root.go

```diff
@@ -94,21 +94,21 @@ func NewTrustedRootFromProtobuf(protobufTrustedRoot *prototrustroot.TrustedRoot)
 		return nil, err
 	}
 
-	trustedRoot.tsaCertAuthorities, err = ParseCertificateAuthorities(protobufTrustedRoot.GetTimestampAuthorities())
+	trustedRoot.timestampingAuthorities, err = ParseCertificateAuthorities(protobufTrustedRoot.GetTimestampAuthorities())
 	if err != nil {
 		return nil, err
 	}
 
-	trustedRoot.ctLogAuthorities, err = ParseTlogAuthorities(protobufTrustedRoot.GetCtlogs())
+	trustedRoot.ctLogs, err = ParseRekorLogs(protobufTrustedRoot.GetCtlogs())
 	if err != nil {
 		return nil, err
 	}
 
 	return trustedRoot, nil
 }
 
-func ParseTlogAuthorities(tlogs []*prototrustroot.TransparencyLogInstance) (tlogAuthorities map[string]*TlogAuthority, err error) {
-	tlogAuthorities = make(map[string]*TlogAuthority)
+func ParseRekorLogs(tlogs []*prototrustroot.TransparencyLogInstance) (rekorLogs map[string]*TransparencyLog, err error) {
```

Comment:
Should this function be `ParseTransparencyLogs` instead? It would help readability for something like line 102, where you assign what looks like "rekor" to "ctLogs"

Reply:
Yeah, good point, I'll change that!